debugger/Makefile

```diff
@@ -25,7 +25,7 @@ CAMLRUN ?= $(ROOTDIR)/boot/ocamlrun
 CAMLYACC ?= $(ROOTDIR)/yacc/ocamlyacc$(EXE)
 
 CAMLC=$(BEST_OCAMLC) -g -nostdlib -I $(ROOTDIR)/stdlib
-COMPFLAGS=$(INCLUDES) -absname -w +a-4-9-41-42-44-45-48 -warn-error A \
+COMPFLAGS=$(INCLUDES) -absname -w +a-4-9-41-42-44-45-48-68 -warn-error A \
```

Comment:
This is doesn't feel like the right fix. As shown by your fixes, the new warning is potentially invasive on some code bases, it is thus better to let the choice to users for now, by making the warning disabled by default.

The list of default warnings is named `defaults_w` in `util/warnings.ml`.

Reply:
I tried `defaults_w` it still gave me the same error
```
../boot/ocamlrun ../ocamlc -strict-sequence -absname -w +a-4-9-41-42-44-45-48 -g -warn-error A -bin-annot -nostdlib -safe-string -strict-formats   -o stdlib__pervasives.cmo -c pervasives.ml
File "/home/anukriti/ocaml/stdlib/pervasives.ml", line 1:
Error (warning 68): Cannot find interface file
```

Reply:
Ah, yes. Sorry, I was mistaken. Removing explicitly the flag is needed an time an executable is generated. This yet another reason to remove the warning from the default flags (while keeping your changes to the compiler flags).

Reply:
Testsuite files such as `w47_inline.ml`, `deprecated_module_assigment`, `w58.ml`, etc still gives the warning because of which travis is failing